packages/deployments/contracts/contracts/messaging/RootManager.sol

```diff
@@ -265,4 +538,32 @@ contract RootManager is ProposedOwnable, IRootManager, WatcherClient, DomainInde
 
     return (_aggregateRoot, _count);
   }
+
+  /**
+   * @notice Watcher can set the system in slow mode.
+   * @dev Clear the proposed aggregate root.
+   */
+  function activateSlowMode() external onlyWatcher onlyOptimisticMode {
+    optimisticMode = false;
+
+    delete proposedAggregateRoot;
+    emit SlowModeActivated();
```

Comment:
we should make sure to emit the watcher that called this in the event

Reply:
👀

Reply:
Should it be indexed too @LayneHaber ?

Reply:
yeah you know I love me an opinionated event topic lets do it